db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
+		bonus bDoubleRate, 25;
```

Comment:
It comes implicit with the "activate it at that level" part and is present in other items such as sidewinder card and chick hat, both of them doesn't have that effect on aegis script, but it's also applied to work. It is exactly the same, if this effect is wrong, then all the items with the same effect also are.